src/cdef.rs

```diff
@@ -44,20 +46,20 @@ bitflags! {
 }
 
 wrap_fn_ptr!(pub unsafe extern "C" fn cdef(
-    dst_ptr: *mut DynPixel,
+    dst_ptr: *const Rav1dPictureDataComponent,
+    dst_offset: usize,
     stride: ptrdiff_t,
     left: *const [LeftPixelRow2px<DynPixel>; 8],
-    top_ptr: *const DynPixel,
-    bottom_ptr: *const DynPixel,
+    top_ptr: *const DisjointMut<AlignedVec64<u8>>,
+    top_offset: usize,
+    bottom_ptr: *const PicOrBuf<'_, AlignedVec64<u8>>,
+    bottom_offset: usize,
     pri_strength: c_int,
     sec_strength: c_int,
     dir: c_int,
     damping: c_int,
     edges: CdefEdgeFlags,
     bitdepth_max: c_int,
-    _dst: *const FFISafe<Rav1dPictureDataComponentOffset>,
-    _top: *const FFISafe<CdefTop>,
-    _bottom: *const FFISafe<CdefBottom>,
 ) -> ());
```

Comment:
We can't change this signature because it's used by asm.  I guess it's not used by arm asm yet, but that may change, and it is used by x86 asm.  So all of the changed arguments (the offsets) need to go at the end, or nowhere at all.  I'm pretty sure this is why CI is failing.